ts/IterableInbox.tsx

```diff
@@ -81,13 +95,12 @@ const IterableInbox = ({
             <Text style={styles.headline}>
                {customizations.navTitle ? customizations.navTitle : defaultInboxTitle}
             </Text>
-            { messages.length ?
+            { rowViewModels.length ?
                <IterableInboxMessageList 
-                  messages={messages}
-                  messageListItemLayout={messageListItemLayout}
+                  rowViewModels={rowViewModels}
                   customizations={customizations}
-                  //deleteMessage={(id: string) => deleteMessage(id, messages)}
-                  handleMessageSelect={(id: string, index: number) => handleMessageSelect(id, index, messages)}
+                  deleteMessage={(id: string, index: number) => deleteMessage(id, index, rowViewModels)}
```

Comment:
`inAppMessageId` and `index` etc. etc. you already know

Reply:
Addressed.